test/integration/controller/lifecycle/resources/crd-resources.gardener.cloud_managedresources.yaml

```diff
@@ -240,4 +240,4 @@ spec:
     served: true
     storage: true
     subresources:
-      status: {}
+      status: {}
```

Comment:
Some other extensions are generating these CRDs based on the vendored version of gardener/gardener. See https://github.com/gardener/gardener-extension-provider-aws/blob/f9edf4533806c8885bdeaa178aba6dea09b98eaf/example/doc.go#L15

You could do the same instead of maintaining them manually.

Reply:
I added a new subtask to https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/issues/2. When the provider extensions tackle the above described issue, then we can also follow their path.